config/software/bash.rb

```diff
@@ -28,8 +29,17 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  # On freebsd, you have to force static linking, otherwise the executable
+  # will link against the system ncurses instead of ours.
+  if freebsd?
+    configure_args = ["--enable-static-link"]
+  else
+    configure_args = []
+  end
+
   configure_command = ["./configure",
-                       "--prefix=#{install_dir}/embedded"]
+                       "--prefix=#{install_dir}/embedded",
+                       configure_args]
```

Comment:
Same things I recommended in the git software def in: https://github.com/chef/omnibus-software/pull/435#discussion-diff-32273638